bundles/net.tourbook.common/src/net/tourbook/common/util/ColumnManager.java

```diff
@@ -1441,8 +1441,8 @@ private void fillHeaderCtxMenu_40_Columns(final Menu contextMenu) {
        */
       final ArrayList<ColumnDefinition> allColumns = getRearrangedColumns();
 
-      final ArrayList<ColumnDefinition> displayedColumns = new ArrayList<>();
-      final ArrayList<ColumnDefinition> notDisplyedColumns = new ArrayList<>();
+      final ArrayList<ColumnDefinition> displayedColumns = new ArrayList<ColumnDefinition>();
```

Comment:
Isn't that extraneous ? See rule [2293](https://rules.sonarsource.com/java/RSPEC-2293)

Reply:
> Isn't that extraneous ? See rule [2293](https://rules.sonarsource.com/java/RSPEC-2293)

I didn't touch this code actually.
on top of that you and wolfgang should talk because he is advocating to have the type in <> and you are advocating for the reverse.

Reply:
Oh yes, that's what I don't like when doing PRs with branches that are not up to date with main because then you don't know if the file change is in the PR or because the destination branch is not updated with the latest main, really annoying.

Well, if @wolfgang-ch prefers to have the type in both <> then I suppose we should do that, I was not aware of it.
Personnally, i don't see the purpose because it's redundant to have it defined twice and would prefer to follow the Java Standard practice rule. But that's my opinion

Reply:
> Oh yes, that's what I don't like when doing PRs with branches that are not up to date with main because then you don't know if the file change is part of the PR or if it's appearing because the destination branch is not updated with the latest main, really annoying.
> I've had cases where I modified 2 files but yet more than 50 files appear in the PR because of that. When going back to the PR then we dont know what was intended to be modified and it creates a bunch of "noise" and possible confusion and at times headaches
> 
> Well, if @wolfgang-ch prefers to have the type in both <> then I suppose we should do that, I was not aware of it.
> Personally, i don't see the purpose because it's redundant to have it defined twice and would prefer to follow the Java Standard practice rule. But that's just my opinion.

Agree with you, especially as the save action preference mentionned in the MTB standard setting says specifically "Remove redundant type arguments" which exactly does what you are advocating. But apparently that settings doesn't behave that way on wolfgang setup. 
At this point I'am oblige to revert back, on my code, with "ctrl+z" to prevent that "save action" rule  to remove the type !!....But ok, at this point I just want the "custom tracks" to be finished/merged in Main, so I can start something else.

Reply:
I personally don't mind if we need to use diamond operator or not as longs as it's consistent.

Reply:
> you and wolfgang should talk because he is advocating to have the type in <> and you are advocating for the reverse( no type in <>).

I cannot remember advocating to have the type in the <> in the constructor, some days ago I just complained that the yellow flag should not appear, which is the case when you have an empty <> in the constructor. 

As I remember the empty <> was missing in the constructor.

I found the discussion https://github.com/wolfgang-ch/mytourbook/pull/286#discussion_r559177528

Reply:
> > you and wolfgang should talk because he is advocating to have the type in <> and you are advocating for the reverse( no type in <>).
> 
> I cannot remember advocating to have the type in the <> in the constructor, some days ago I just complained that the yellow flag should not appear, which is the case when you have an empty <> in the constructor.
> 
> As I remember the empty <> was missing in the constructor.
> 
> I found the discussion [#286 (comment)](https://github.com/wolfgang-ch/mytourbook/pull/286#discussion_r559177528)

@wolfgang-ch , sorry, it seems I misunderstood your review I thought it was about specifying the type, now I got it. it was simply the missing <> in the constructor which was the issue.